packages/realm-web/src/OAuth2Helper.test.ts

```diff
@@ -41,7 +41,7 @@ describe("OAuth2Helper", () => {
                             userAuth: "our-little-secret",
                         }),
                     );
-                });
+                }, 10);
```

Comment:
Why 10? It seems like omitting a number here would be closest to the old code.

Reply:
I honestly can't remember if there was a good reason that this was added, I'll revert.